Bottom: 21fea53f44a5b6c66ba534813c62f2a3e45743a8
Top:    106095f6dca1a1342407b7c5dac0598e39b47c14
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-23 17:37:23 +0000

Add hid_draw.h to Makefile.am


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 269ee78..602ebaa 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -79,6 +79,7 @@ PCB_SRCS = \
 	heap.c \
 	heap.h \
 	hid.h \
+	hid_draw.h \
 	insert.c \
 	insert.h \
 	intersect.c \
